apache/trafficcontrol
# Add Monitor BufferStatProcessing config, to reduce CPU

#### What does this PR do?

Add Monitor BufferStatProcessing config, to reduce CPU

#### Which TC components are affected by this PR?


#### What is the best way to verify this PR?

1. Add a `traffic_monitor.cfg` key `buffer_stat_processing` set to `true`
2. Change `traffic_monitor.cfg` `stat_flush_interval_ms` to `1000`
3. Verify monitor behaves correctly. The behavior should not change, except to improve poll times and/or reduce CPU.
4. If want to verify the code is executing, enable Info logging and look in the log for messages `StatHistoryManager timer expired ({in,after} read): processed n results in m`

#### Check all that apply
